lib/iris/cube.py

```diff
@@ -3650,6 +3650,23 @@ def __itruediv__(self, other):
     __pow__ = iris.analysis.maths.exponentiate
     # END OPERATOR OVERLOADS
 
+    def _trim_ancils(self, dims_to_collapse):
+        reduced_cube = self.copy()
+
+        # Collapse any ancillary variables that span the dimension(s) being collapsed
```

Comment:
"Collapse any ancillary variables that span ..."
Should read "**Remove** any ancillary variables that span ..."

_+1 similar, immediately below_